### PR TITLE
Custom separators for action types

### PR DESCRIPTION
**Pull request by roboslone**
Solves #165.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -44,7 +45,7 @@ export default function promiseMiddleware(config = {}) {
        * @returns {object} action
        */
       const getAction = (newPayload, isRejected) => ({
-        type: `${type}_${isRejected ? _REJECTED : _FULFILLED}`,
+        type: `${type}${promiseTypeSeparator}${isRejected ? _REJECTED : _FULFILLED}`,
```
Comment (pburtchaell) [inline src/index.js]: Perhaps `[type, isRejected ? _REJECTED' : _FULFILLED].join(promiseTypeSeparator)` would be better.

**Comment by pburtchaell** [comment]
Hey @roboslone, what's your status here?

**Comment by roboslone** [comment]
I've been busy on weekdays, I'll try to fix this on weekend.

**Comment by pburtchaell** [comment]
Thank you! The last step is just to add documentation.

**Comment by roboslone** [comment]
Whoops, didn't think about docs. I'll get to that as soon as I can.

**Comment by roboslone** [comment]
@pburtchaell, I've added another guide for separators, but I'm not sure if separator can be changed locally (like custom suffixes). Could you take a look?

**Comment by pburtchaell** [comment]
Great! Thank you @roboslone. I'm busy this week, but I'll give this a final review as soon as possible.

**Comment by roboslone** [comment]
@pburtchaell, any progress here?

**Comment by pburtchaell** [comment]
Sorry for the delay. I've been working overtime at work. I'll get to it as soon as possible!

**Comment by roboslone** [comment]
@pburtchaell any chance for this PR to be merged this week? I really miss this functionality.

**Pull request merged**